File: Models/ExamModel.cs
using System;

namespace ExamPortal.Models
{
    public class ExamModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public int Duration { get; set; }
        public int QuestionsNumber { get; set; }
        public DateTime StartDate { get; set; }
    }
}

File: ClientApp/src/app/models/exam.model.ts
export interface ExamModel {
    id: number;
    title: string;
    duration: number;
    questionsNumber: number;
    startDate: string;
    available: boolean;
}

export interface ExamViewModel {
  no: number;
  title: string;
  duration: number;
  questionsNumber: number;
  startDate: string;
  available: boolean;
}